feature/AT-UN-UI-Edit-Admin-Course

acones96 opened the pull request.

Commit:
Implement routes and controllers

Commit:
Delete first changes and add /admin

Commit:
Change router & controller path and hbs develop

Commit:
Change routes, controller, api and start hbs

at-internship commented on src/controllers/admin.controller.js:
```diff
@@ -47,14 +47,18 @@ adminCtrl.addCourse = async(req, res) => {
+adminCtrl.updateCourse = async(req, res) => {
     // Redirect
+    const { title, description, status, category, img } = req.body; 
+    await Course.findByIdAndUpdate(req.params.id, { title, description, status, category, img });
```
Where is this function located?

at-internship commented on src/controllers/admin.controller.js:
```diff
@@ -47,14 +47,18 @@ adminCtrl.addCourse = async(req, res) => {
     console.log("--> adminCtrl.renderEditCourseForm");
-    res.render("admin/course/edit-course");
+
+    const course = await Course.findById(req.params.id).lean();
```
Where is this function located?

Commit:
miss /dev hbs

Commit:
delete mongoose

Commit:
Prove

Commit:
Controller, routs, api-service changes

at-internship commented:
this branch sould be `feature/AT-UN-UI-Edit-Admin-Course`

Commit:
Changes at edit-course.hbs, server and controllers

We implement some helpers at server.js,  modify the hbs and controllers functions

Commit:
Last changes

Commit:
Fix Update
